securedrop/source_app/main.py

```diff
@@ -91,6 +92,7 @@ def lookup():
             codename=g.codename,
             replies=replies,
             flagged=g.source.flagged,
+            new_user=session['new_user'],
```

Comment:
this should get `session('new_user', None)` so that if it's not present it won't throw a `KeyError`.